content/en/docs/zero-code/js/configuration.md

```diff
@@ -45,21 +45,50 @@ For example, to only enable the `env` and `host` detectors, you can set:
 OTEL_NODE_RESOURCE_DETECTORS=env,host
 ```
 
-### Excluding instrumentation libraries
+## Excluding instrumentation libraries
 
 By default, all
 [supported instrumentation libraries](https://github.com/open-telemetry/opentelemetry-js-contrib/blob/main/metapackages/auto-instrumentations-node/README.md#supported-instrumentations)
-are enabled, but you can use the environment variable
-`OTEL_NODE_ENABLED_INSTRUMENTATIONS` to enable only certain instrumentations by
-providing a comma-separated list of the instrumentation library names without
-the `@opentelemetry/instrumentation-` prefix.
+are enabled, but you can use environment variables to enable or disable specific
+instrumentations.
+
+### Enable specific instrumentations
+
+Use the environment variable `OTEL_NODE_ENABLED_INSTRUMENTATIONS` to enable only
+certain instrumentations by providing a comma-separated list of the
+instrumentation library names without the `@opentelemetry/instrumentation-`
+prefix.
 
 For example, to enable only
-[@opentelemetry/instrumentation-http](https://github.com/open-telemetry/opentelemetry-js/tree/main/packages/opentelemetry-instrumentation-http)
+[@opentelemetry/instrumentation-http](https://github.com/open-telemetry/opentelemetry-js/tree/main/experimental/packages/opentelemetry-instrumentation-http)
 and
 [@opentelemetry/instrumentation-express](https://github.com/open-telemetry/opentelemetry-js-contrib/tree/main/plugins/node/opentelemetry-instrumentation-express)
 instrumentations:
 
 ```shell
 OTEL_NODE_ENABLED_INSTRUMENTATIONS="http,express"
 ```
+
+### Disable specific instrumentations
+
+Use the environment variable `OTEL_NODE_DISABLED_INSTRUMENTATIONS` to keep the
+fully enabled list and only disable certain instrumentations by providing a
+comma-separated list of the instrumentation library names without the
+`@opentelemetry/instrumentation-` prefix.
+
+For example, to disable only
+[@opentelemetry/instrumentation-fs](https://github.com/open-telemetry/opentelemetry-js-contrib/tree/main/plugins/node/instrumentation-fs)
+and
+[@opentelemetry/instrumentation-grpc](https://github.com/open-telemetry/opentelemetry-js/tree/main/experimental/packages/opentelemetry-instrumentation-grpc)
+instrumentations:
+
+```shell
+OTEL_NODE_DISABLED_INSTRUMENTATIONS="fs,grpc"
+```
+
+{{% alert title="Notes" color="info" %}}
+
+`OTEL_NODE_ENABLED_INSTRUMENTATIONS` and `OTEL_NODE_DISABLED_INSTRUMENTATIONS`
+are mutually exclusive. If both are set, all instrumentations are disabled.
```

Comment:
Similar Q to above about the behaviour if both are specified.

Reply:
Updated!